tests/Utilities/Plugin_Request_Utility_Tests.php

```diff
@@ -97,6 +100,7 @@ function( $checks ) {
 		$runner  = Plugin_Request_Utility::get_runner();
 
 		unset( $_SERVER['argv'] );
+		$wp_actions['muplugins_loaded'] = $muplugins_loaded;
```

Comment:
This feels like this should in the `tear_down` method

Reply:
See my reply in https://github.com/10up/plugin-check/pull/100#discussion_r1145234889.